spec/requests/shop_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe "Shop API" do
   include ShopWorkflow
+  include RSpec::Rails::RequestExampleGroup # to have get/post methods in not controller
```

Comment:
Given that we have to bend RSpec to our will here a bit, and it's going against convention, I wonder whether it would make more sense to turn these into request specs. What do you think?